src/veriables.scss

```diff
@@ -0,0 +1,3 @@
+$main-text-color: #060b35;
+$main-button-color: #00acdc;
+$main-font-famaly: Roboto;
```

Comment:
There is a typo in the variable name '$main-font-famaly'. It should be '$main-font-family'. Update this variable name and ensure all references in your SCSS files are consistent.